lib/ansible/utils/encrypt.py

```diff
@@ -68,7 +68,7 @@ class BaseHash(object):
     algo = namedtuple('algo', ['crypt_id', 'salt_size', 'implicit_rounds', 'salt_exact'])
     algorithms = {
         'md5_crypt': algo(crypt_id='1', salt_size=8, implicit_rounds=None, salt_exact=False),
-        'bcrypt': algo(crypt_id='2a', salt_size=22, implicit_rounds=None, salt_exact=True),
+        'bcrypt': algo(crypt_id='2b', salt_size=22, implicit_rounds=None, salt_exact=True),
```

Comment:
make it a try/except test to check its available and then assign the value depending on that

Reply:
Add a test case and changelog entry for this change.

Reply:
how would you want the check to look im not sure, and not sure on how to add the test for it. can i get an assist please?

Reply:
author closed, I guess he gave up.

In any case the try/except change i'm suggesting has with attempting to use the newer crypt_id but defaulting to the old one if that fails, making it a backwards compatible change.